ui/core-react/src/core-react/contextmenu/ContextMenu.scss

```diff
@@ -221,6 +221,8 @@ $uicore-menu-top-bottom-padding: calc(#{$uicore-menu-padding} * 0.6667);
         font-size: calc(var(--iui-font-size-2) * 0.5);
         line-height: var(--iui-size-l);
         text-align: center;
+        position: relative;
```

Comment:
From what I see, most of the tweaks from this class were for the previous "webfont" mode, I think we can simplify all of this now by removing everything but 
```scss
{
  margin-left: $uicore-unbordered-padding;
  margin-top: var(--iui-size-3xs);
}
```
and get the same result with less "What's going on" when looking at this class.

Reply:
Looks good in storybook with that simplified styling. Changes pushed